pkgs/by-name/gi/git-butler/package.nix

```
@@ -0,0 +1,71 @@
+{ lib, stdenv, fetchurl, fetchzip, appimageTools, undmg }:
+let
+  pname = "git-butler";
+  version = "0.11.7";
+  build_version = "889";
+
+  src = {
+    x86_64-darwin = fetchurl {
+      url =
+        "https://releases.gitbutler.com/releases/release/${version}-${build_version}/macos/x86_64/GitButler_${version}_x64.dmg";
+      hash = "sha256-5mpcrwZ+BogmbEE/jAlOFvo5KA58IGV/peQFxveY9o8=";
+    };
+    aarch64-darwin = fetchurl {
+      url =
+        "https://releases.gitbutler.com/releases/release/${version}-${build_version}/macos/aarch64/GitButler_${version}_aarch64.dmg";
+      hash = "sha256-uej3EEbG21EKdjRHJE9cgvRrU3dZclf9KQt7L6Ggz0U=";
+    };
+    x86_64-linux = fetchzip {
+      url =
+        "https://releases.gitbutler.com/releases/release/${version}-${build_version}/linux/x86_64/git-butler_${version}_amd64.AppImage.tar.gz";
+      hash = "sha256-XuqyhaE1gi04m7F1V/Ca82aUpL8H8H6fnnPTiEX9oBY=";
+    };
+  }.${stdenv.system} or (throw "Unsupported system: ${stdenv.system}");
+
+  meta = {
+    description =
+      "A Git client for simultaneous branches on top of your existing workflow.";
+    mainProgram = "git-butler";
+    homepage = "https://gitbutler.com/";
+    changelog =
+      "https://discord.com/channels/1060193121130000425/1183737922785116161";
+    platforms = [ "aarch64-darwin" "x86_64-darwin" "x86_64-linux" ];
```

```suggestion
    platforms = lib.platforms.unix;
```
Is there any specific UNIX platform on which you know it not to work?

---

I haven't tested on aarch64 for linux. I don't know if the appimage have a valid usage for this arch